numba/targets/arraymath.py

```diff
@@ -1413,6 +1413,33 @@ def np_tril_impl_multi(m, k=0):
     else:
         return np_tril_impl_multi
 
+@overload(np.tril_indices)
+def np_tril_indices(n, k=0, m=None):
+
+    # we require integer arguments, unlike numpy
```

Comment:
You might consider putting all the common type-checking for this and the one below (`triu_indices`)  in a re-usable function.

Reply:
Done